src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
I'm open to that. 

The only consideration I'd want to chat about is the extent to which we want to encourage modularity VS sharing - between transforms either in JS or CSS.

By having JS and accompanying CSS in same folder makes it maybe a little easier to think of them in pair-wise fashion (each JS has CSS file of same name if needed) rather than thinking of CSS folder as a place you put CSS to be shared between JS transform files.

...though we could outline best practice in readme.md once we decide what those are :)